scripts/gis-material-update/README.md

```diff
@@ -0,0 +1,127 @@
+# Intro
+
+Download and process Haitaton spatial material (laastariaineisto).
```

Comment:
Maybe we should drop the "laastariaineisto" moniker, since we are trying to make the material production-ready.

Reply:
Didn't drop it yet, but rephrased.